TicTacToe/TicTacToe.cs

```diff
@@ -4,84 +4,158 @@ namespace TicTacToe
 {
     class Program
     {
+        //variable that determins who's turn it is
         public static string playerTurn = "X";
+
+        //Array that keeps track of places on board
         public static string[][] board = new string[][]
         {
             new string[] {" ", " ", " "},
             new string[] {" ", " ", " "},
             new string[] {" ", " ", " "}
         };
-
         public static void Main()
         {
+            //draws board and runs game while there is no winner or tie
             do
             {
                 DrawBoard();
                 GetInput();
-
-            } while (!CheckForWin() && !CheckForTie());
-
-            // leave this command at the end so your program does not close automatically
+            } while  (!CheckForWin()&&!CheckForTie());   
+            // keeps program from closing automaticly
             Console.ReadLine();
         }
 
+        //gets players move in terms of row and column number
         public static void GetInput()
         {
             Console.WriteLine("Player " + playerTurn);
-            Console.WriteLine("Enter Row:");
-            int row = int.Parse(Console.ReadLine());
             Console.WriteLine("Enter Column:");
-            int column = int.Parse(Console.ReadLine());
-        }
-
+            int column = Convert.ToInt16(Console.ReadLine()) - 1;
+            Console.WriteLine("Enter Row:");
+            int row = Convert.ToInt16(Console.ReadLine()) - 1;
+            PlaceMark(row,column);
+        }   
         public static void PlaceMark(int row, int column)
```

Comment:
This method is called PlaceMark, but you are checking for win or tie? seems a little out of place.